charts/redpanda/statefulset.go

```diff
@@ -212,13 +216,21 @@ func StatefulSetVolumes(dot *helmette.Dot) []corev1.Volume {
 		},
 	}...)
 
+	if len(values.Listeners.TrustStores(&values.TLS)) > 0 {
+		volumes = append(
+			volumes,
+			values.Listeners.TrustStoreVolume(&values.TLS),
```

Comment:
Would that change work?
```suggestion
	if vols := values.Listeners.TrustStores(&values.TLS); len(vols) > 0 {
		volumes = append(
			volumes,
			vols,
```

Reply:
There's only a single volume that's returned so I couldn't do it that way (and it's two different method calls). I've change it so that the truststorevolume returns a pointer that's nil when no truststores are configured.